src/main/java/racing/ViewConstant.java

```diff
@@ -0,0 +1,13 @@
+package racing;
```

Comment:
view에서만 사용되는 클래스는 view 패키지로 이동하면 좋을 것 같아요.
```suggestion
package racing.view;
```